packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -9,28 +9,28 @@
                 </div>
             </div>
             <div class="flex-initial max-w-md pr-2">
-                <p class="text-left text-sm font-bold">
+                <p class="text-left text-sm font-bold w-5/6 mobileLandscape:w-full">
                     {{#t}}Firefox accounts will be renamed Mozilla accounts on Nov 1{{/t}}
                 </p>
-                <p class="text-left text-xs">
+                <p class="text-left text-xs w-5/6 mobileLandscape:w-full">
                     {{#t}}You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
-                    <span class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
+                    <span role="link" class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
                 </p>
             </div>
         </div>
         {{/showPrelaunch}}
         {{#showPostlaunch}}
         <div className="flex"/>
             <div>
-                <p class="text-sm font-bold">
+                <p class="text-sm font-bold w-5/6 mobileLandscape:w-full">
                     {{#t}}We’ve renamed Firefox accounts to Mozilla accounts. You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
                     <span role="link" class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
                 </p>
             </div>
         </div>
         {{/showPostlaunch}}
     </div>
-    <div class="absolute w-10 right-0 top-0 pt-3 start-0">
+    <div class="absolute w-10 top-0 rtl:left-0 ltr:right-0 pt-3">
         <img id="close-brand-banner" class="cursor-pointer" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiI+CiAgPGc+CiAgICA8cGF0aCBmaWxsPSJyZ2JhKDEyLCAxMiwgMTMsIC44KSIgZD0iTTkuNDE0IDhsNS4yOTMtNS4yOTNhMSAxIDAgMDAtMS40MTQtMS40MTRMOCA2LjU4NiAyLjcwNyAxLjI5M2ExIDEgMCAwMC0xLjQxNCAxLjQxNEw2LjU4NiA4bC01LjI5MyA1LjI5M2ExIDEgMCAxMDEuNDE0IDEuNDE0TDggOS40MTRsNS4yOTMgNS4yOTNhMSAxIDAgMDAxLjQxNC0xLjQxNHoiLz4KICA8L2c+Cjwvc3ZnPgo=" alt="{{#t}}Close Banner{{/t}}" />
     </div>
```

Comment:
This is an edgecase also since I don't think there's any devices of this size but we could add more padding on the right to prevent this overlap (keep resizing downward until you see it) or not use `position: absolute` and use flexbox instead:
<img width="400" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/866f802e-6f5d-4a0a-b252-9607aededed7">

Reply:
I've switched to using flexbox. It's much cleaner! Thanks for the suggestion 👍